src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/DotNetCoreProjectCompatibilityDetector.cs

```diff
@@ -255,7 +255,7 @@ public int OnAfterBackgroundSolutionLoadComplete()
 
         private static async Task<CompatibilityLevel> GetProjectCompatibilityAsync(UnconfiguredProject project, VersionCompatibilityData compatData)
         {
-            if (project.Capabilities.AppliesTo(ProjectCapability.CSharpOrVisualBasicOrFSharp))
+            if (project.Capabilities.AppliesTo(ProjectCapability.CSharpOrVisualBasicOrFSharp + " & " + ProjectCapability.PackageReferences))
```

Comment:
this will result in:
```
CSharp | VisualBasic | FSharp & PackageReference
```

Not sure what order of operations is, but can we parenthesis these to make this clear?

```
(CSharp | VisualBasic | FSharp) & PackageReference
```

Reply:
Updated - thanks for catching.